apps/back/package.json

```
@@ -12,6 +12,8 @@
     "node": "v18.11.0"
   },
   "dependencies": {
+    "@types/body-parser": "^1.19.2",
+    "body-parser": "^1.20.2",
```

Suggest to use fixed version for dependencies, remove "^".
```suggestion
    "body-parser": "1.20.2",
```